src/main/java/com/minecolonies/entity/EntityCitizen.java

```diff
@@ -1230,7 +1232,8 @@ private void sendChat(String msg)
         statusMessages.put(msg, ticksExisted);
 
         LanguageHandler.sendPlayersMessage(
-                ServerUtils.getPlayersFromUUID(worldObj, getColony().getPermissions().getMessagePlayers()),
+                colony.getMessageEntityPlayers(),
+                //TODO does this need to go through the LanguageHandler#format?
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1135)